sources/packages/backend/apps/api/src/services/application-bulk-withdrawal/application-bulk-withdrawal-import-validation.service.ts

```diff
@@ -0,0 +1,149 @@
+import { BadRequestException, Injectable } from "@nestjs/common";
+import {
+  ValidationResult,
+  ValidationContext,
+  ValidationContextTypes,
+  ApplicationBulkWithdrawalImportBusinessValidationModel,
+  ApplicationWithdrawalValidationResult,
+} from "./application-bulk-withdrawal-import-business-validation.models";
+import { validateSync, ValidationError } from "class-validator";
+import { plainToClass } from "class-transformer";
+import { flattenErrors } from "../../utilities/class-validation";
+import { CustomNamedError } from "@sims/utilities";
+import {
+  APPLICATION_BULK_WITHDRAWAL_VALIDATION_CRITICAL_ERROR,
+  APPLICATION_WITHDRAWAL_VALIDATION_ERROR,
+} from "../../constants";
+import { ApiProcessError } from "../../types";
+import { ApplicationBulkWithdrawalValidationResultAPIOutDTO } from "../../route-controllers/student-scholastic-standings/models/student-scholastic-standings.dto";
+
+/**
+ * Executes the validations on an offering model.
+ */
+@Injectable()
+export class ApplicationBulkWithdrawalImportValidationService {
+  /**
+   * Validate offering models and provide the result for every model.
+   * @param businessValidationModels application bulk withdrawal business validation models to be validated.
+   * @param silently if true, no exception is generated case the validation fails
+   * and the success or failure can be determined from the result object.
+   * @returns validation result or an exception in the case of a failed validation
+   * combined with the silently parameter defined as false.
+   */
+  validateApplicationBulkWithdrawalModels(
+    businessValidationModels: ApplicationBulkWithdrawalImportBusinessValidationModel[],
+    silently = false,
```

Comment:
what is the benefit of this variable, I see this method is only used once

Reply:
@ann-aot I guess that it is supposed to work similarly to offerings where we use it to allow the method to perform a "validation-only" or the "real" import, but I believe that it does not completely follow the same idea, hence a change is needed.

Reference code (please note that it is calling a controller service).
https://github.com/bcgov/SIMS/blob/81923206ba137731f7fd4e3e6f1776369f8a6898/sources/packages/backend/apps/api/src/route-controllers/education-program-offering/education-program-offering.institutions.controller.ts#L551-L555

Reply:
Removed it.